src/routers/products.js

```diff
@@ -0,0 +1,22 @@
+const productsRouter = require('express').Router();
+const { ProductsController } = require('../controllers');
+const {
+  createProductPayloadValidator,
+  updateProductPayloadValidator
+} = require('../middlewares/validators');
+
+const productsController = new ProductsController();
+productsRouter.post(
+  '/',
+  createProductPayloadValidator,
+  productsController.createProduct
+);
+productsRouter.put(
+  '/revision/:id',
```

Comment:
endpoint must be just /:id. Not "revision", "stock"... get(/), post(/), put(/:id), delete(/:id)

Reply:
Fixed